src/lib/Components/Bidding/Screens/__tests__/Registration-tests.tsx

```diff
@@ -661,6 +661,34 @@ describe("when pressing register button", () => {
       needsIdentityVerification: false,
     })
   })
+
+  it("displays the pending result with needsIdentityVerification: true when the sale requires identity verification", () => {
```

Comment:
I a bit confused that the docstring says that it displays a "pending result" yet one of the assertions include `RegistrationStatus.RegistrationStatusComplete`? Should one of these be updated or can you clarify?

Reply:
This was just left as-is after copy-and-pasting. I'll update.

```suggestion
  it("displays the completed result when the bidder is not verified but qualified for bidding", () => {
```